### ISSUE TITLE
Detector - Detector process timing out and not releasing file locks

### ISSUE THREAD

**Issue by steviecoaster** [chocolatey/chocolatey-licensed-issues#59]
### What You Are Seeing?

The package installation is falling over while the detector service is working with the installer binaries.

### What is Expected?
The package installs successfully
### How Did You Get This To Happen? (Steps to Reproduce)
run `choco install _packagename`
### Output Log
```
Chocolatey timed out waiting for the command to finish. The timeout 
specified (or the default value) was '60' seconds. Perhaps try a 
higher `--execution-timeout`? See `choco -h` for details.

[..snip..]

ERROR: The process cannot access the file because it is being used by another process
```

**Comment by steviecoaster** [comment]
https://gitlab.com/chocolatey/collaborators/choco-licensed/issues/257

**Comment by ferventcoder** [comment]
This has been fixed and will be released with Chocolatey Licensed Extension v2.0.0.
